meteor/lib/logging.ts

```diff
@@ -69,6 +59,7 @@ if (Meteor.isServer) {
 } else {
 	const getLogMethod = (type) => {
 		return (...args) => {
+			// TODO: Should we also send errors and warnings to the server?
```

Comment:
As a rule, our Front-end code doesn't really log anything anywhere. Any errors that happen, happen either in React, Meteor or are uncaught exceptions, so I don't think there's any point. You shouldn't be using the logger in FE code in the first place.